### ISSUE TITLE
Wifi with tdslite

### ISSUE THREAD

**Issue by redarc89** [mustafakemalgilor/tdslite#8]
Hello,

Can we use tdslite with the WiFi library in Arduino IDE ?
I use WiFi and I need tu use tdslite but I can't une ethernet and in whole example a see you use ethernet library.

**Comment by mustafakemalgilor** [comment]
@redarc89 of course, take a look at the tests/sketches/esp/esp.cpp as an example. The library supports all wifi/ethernet libraries compatible with the Arduino's EthernetClient interface.
